tests/next_tests/integration_tests/feature_tests/ffront_tests/test_gt4py_builtins.py

```diff
@@ -56,7 +56,12 @@ def testee(edge_f: cases.EField) -> cases.VField:
     out = cases.allocate(unstructured_case, testee, cases.RETURN)()
 
     v2e_table = unstructured_case.offset_provider["V2E"].table
-    ref = np.max(inp.ndarray[v2e_table], axis=1)
+    ref = np.max(
+        inp.asnumpy()[v2e_table],
+        axis=1,
+        initial=np.iinfo(np.int32).min,
```

Comment:
It's not wrong but `np.iinfo(np.int32).min` feels a bit arbitrary. What do you think about using `np.min(inp.asnumpy())`?